Documentation/WritingDocsOfficial/HowYouCanHelp.rst

```diff
@@ -83,189 +70,165 @@ cheat sheets.
-*  What advice can you give others?
+*   What gave you the motivation to contribute?
+*   How did you benefit from contributing? For example, what did you learn?
+*   What advice can you give others?
+
 
 .. _how-you-can-help-new-content:
 
-Make Suggestions for New Content
+Make suggestions for new content
 ================================
 
 Sometimes you can directly add content in a manual or create an issue for it.
 
 But, what if you are not sure, where the new content should go or if it should
 be created at all?
 
-If this is the case, either:
-
-*  Raise the question in the #typo3-documentation channel on Slack
-*  Or, add a comment with your suggestion to this general issue:
-   `What content is missing in the documentation? <https://github.com/TYPO3-Documentation/T3DocTeam/issues/85>`__
+If this is the case, raise the question in the `#typo3-documentation`_ channel
+on `Slack`_.
 
 
 .. _how-you-can-help-changelog:
 
-Add Information from the ChangeLog
+Add information from the changelog
 ==================================
 
-.. important::
-
-   This is already being taken care of by the TYPO3 Core Team. For each release,
-   the Changelogs are checked and official documentation is updated. This mostly
-   concerns TYPO3 Explained and the references.
+Each new or changed changelog from the TYPO3 Core creates an issue in the
+`Changelog-to-doc repository`_. The issues are mostly already tagged with the
+according manual which has to be changed. If you want to pick up an issue,
+check if there is already a person assigned. If not leave a comment in the issue
+that you are working on it. This prevents two people working on the same issue.
 
 Regardless of this, if you find something missing, you can add it yourself. Or
-ask in Slack #typo3-documentation if you can help out.
+ask in the `#typo3-documentation`_ Slack channel, if you can help out.
+
 
 .. _how-you-can-help-check-spelling:
 
-Check Spelling
+Check spelling
 ==============
 
 Check spelling for consistency. For example, compare spelling of title
-and headlines to `The TYPO3 Writing Style Guide
-<https://typo3.org/community/teams/content/writing-style-guide/>`__ on typo3.org.
+and headlines to the rules outlined in the :ref:`spelling` chapter.
 
-Is spelling of proper nouns correct?
 
-See :ref:`content-styleguide` for more information, use the online tool
-https://capitalizemytitle.com/ to check title spelling.
+More ideas?
+===========
 
+Do you have more ideas to add to this list? Either add them directly to this
+page using one of :ref:`the two workflows <contribute>` or contact us in the
+`#typo3-documentation`_ channel on `Slack`_.
 
-More Ideas?
-===========
 
-Do you have more ideas to add to this list? Either add them directly to this page using one of
-:ref:`the two workflows <contribute>`
-or `contact us
-<https://typo3.org/community/teams/documentation/#c9886>`__ and make your suggestions.
+..  _@documentation@typo3.social: https://typo3.social/@documentation
```

Comment:
I don't really like exchanging inline links for named ones. The inline links are much closer to html so I think easier to understand for devs. And also when you are reading the source code you can just open the link and don't have to look for it in another place. I think we also suggested somewhere to always use inline links. And we made effords to move away from the named links

Reply:
In this chapter I am using quite some links more than once. I wanted to avoid to add those multiple times. So I opted to move all the links to the end.

Additionally, I like the named links as the rst is better readable IMHO.

Reply:
It introduces more Syntax that we seldom use and have been trying to get rid of in the past years.

Reply:
It is implemented in the guides and what ppl are doing in extensions is their problem. But I think we should document a way to go and stick to it to make it easier for non-rst users to contribute. And I think we documented it here and should stick to that: https://docs.typo3.org/m/typo3/docs-how-to-document/main/en-us/WritingReST/Reference/Content/Links.html#external-links If we want to use different syntax because it is very important then we should also document it there

Reply:
So I should add the same link over and over again on the same page, just to meet some other documentation?

Reply:
$0.02: I can understand both your POV. Especially because I don't really get RST links quite yet, I find those very hard to quickly understand and always shy away from it. Always looks like Perl code to me.

However.

Avoiding redundancy should IMO be the larger issue here, and to me it weighs over "requiring to learn other syntax". I'll have have to face that syntax at some point, and others writing docs probably too. RST already isn't that newbie-friendly...

Reply:
ok, lets keep it then, just don't make it a mass change all over then please